crates/re_types/src/lib.rs

```diff
@@ -74,8 +74,10 @@
 // ---
 
 /// Anything that can be serialized to and deserialized from Arrow data.
-pub trait Loggable {
+pub trait Loggable: Sized {
     type Name;
+    type Item<'a>;
+    type IterItem<'a>: Iterator<Item = Self::Item<'a>>;
```

Comment:
Why not just `Iter`?

Reply:
Just trying to make it more clear that this isn't an iterator of `Loggable`s.